pkg/fanal/analyzer/const.go

```diff
@@ -71,10 +73,11 @@ const (
 	TypeCondaPkg Type = "conda-pkg"
 
 	// Python
-	TypePythonPkg Type = "python-pkg"
-	TypePip       Type = "pip"
-	TypePipenv    Type = "pipenv"
-	TypePoetry    Type = "poetry"
+	TypePythonPkg 	  Type = "python-pkg"
+	TypePip       	  Type = "pip"
+	TypePipenv    	  Type = "pipenv"
+	TypePoetry    	  Type = "poetry"
+	TypePythonGeneric Type = "python"
```

Comment:
looks like we can use [executable](https://github.com/aquasecurity/trivy/blob/d4da83c633a46ad4a61844d8d5502d87b99465a0/pkg/fanal/analyzer/const.go#L99) for new binaries

Reply:
I understand what you meant here now, that all the new binaries could be TypeExecutable.

I use the language specific types in fanal executable.go, to figure out which binary type was detected in case of different binary. It can be localized to executable.go instead and use the common type.

Reply:
I added them as a separate, new group of variables instead in this same file.